api/swagger.yaml

```diff
@@ -4021,6 +4021,53 @@ definitions:
     example:
       Warning: "unable to pin image doesnotexist:latest to digest: image library/doesnotexist:latest not found"
 
+  ImagePlatform:
+    type: "object"
+    x-nullable: true
+    description: |
+      Platform information of the image that the container was created from.
```

Comment:
Should this mention explicitly that this follows the OCI definition of the fields?

(Maybe we need a PR to the image-spec to add an anchor to https://github.com/opencontainers/image-spec/blob/v1.1.0/image-index.md#:~:text=generate%20an%20error.-,platform%20object,-This%20OPTIONAL%20property -- I find myself linking to it constantly with this janky browser-specific syntax :joy:)